tests/cases/conformance/es2018/dynamicImport/importCallExpression2ES2018.ts

```diff
@@ -0,0 +1,17 @@
+﻿// @module: es2018
+// @target: esnext
+// @filename: 0.ts
+export class B {
+    print() { return "I am B"}
+}
+
+// @filename: 2.ts
+// We use Promise<any> for now as there is no way to specify shape of module object

```

Comment:
It is possible to:

```ts
import * as zero from './0'
function foo (x: Promise<typeof zero>) {}
```

but that _requires_ typescript's import elision to behave correctly.